webrtc/RTCCertificate-postMessage.html

```diff
@@ -73,5 +73,28 @@
         assert_throws("InvalidAccessError", () => { new RTCPeerConnection({certificates: [certificate2]}) });
         iframe.remove();
     }, "Check cross-origin created RTCCertificate");
+
+    promise_test(async t => {
+        let certificate1 = await  RTCPeerConnection.generateCertificate({ name: 'ECDSA', namedCurve: 'P-256' });
+        let certificate2 = await  RTCPeerConnection.generateCertificate({ name: 'ECDSA', namedCurve: 'P-256' });
+
+        let url = "resources/RTCCertificate-postMessage-iframe.html";
+        let iframe = await with_iframe(url);
+
+        let promise = new Promise((resolve, reject) => {
+            window.onmessage = (event) => {
+                resolve(event.data);
+            };
+            t.step_timeout(() => reject("Timed out waiting for frame to send back certificate"), 5000);
+        });
+        iframe.contentWindow.postMessage(certificate1, "*");
+        let certificate3 = await promise;
```

Comment:
This seems like a heavy-handed way to create a new object.
Given that RTCCertificate is Serializable, can't it be cloned in a simpler way?
(Note to self: It seems that Serializable is now removed from WebIDL. I don't know what it's called this week.)